loginpage/js/loginpage.js

```diff
@@ -1,21 +1,21 @@
 $(document).ready(function()
 {
-	$.ajax(	"/api/login.json", {
-	    data: { checkLogin: true },
-		dataType: "json",
-		success: function (response) {
-			if(response.loggedIn){
-				$("#status-box").text(response.message);
-				$("#status-box").addClass("error");
-				$("#loginForm").addClass("hidden");
-				$("#logoutForm").removeClass("hidden");
-			}
-		},
-		error: function (xhr, ajaxOptions, thrownError) {
-			$("#status-box").text(thrownError);
-			$("#status-box").addClass("error");
-		},
-	});
+    $.ajax( "/api/login.json", {
+        data: { checkLogin: true },
+        dataType: "json",
+        success: function (response) {
+            if(response.loggedIn){
+                $("#status-box").text(response.message);
+                $("#status-box").addClass("error");
+                $("#loginForm").addClass("hidden");
+                $("#logoutForm").removeClass("hidden");
+            }
+        },
+        error: function (xhr, ajaxOptions, thrownError) {
+            $("#status-box").text(thrownError);
+            $("#status-box").addClass("error");
+        },
+    });
 
     function setRemember(){
```

Comment:
Opening brace should be on a new line

*PHPCodeSnifferBear, severity NORMAL, section `js`.*